22-discovery.ks

```diff
@@ -30,10 +30,7 @@ systemctl enable nm-prepare.service
 echo " * enabling required system services"
 systemctl enable ipmi.service
 systemctl enable foreman-proxy.service
-systemctl enable discovery-setup.path
-systemctl enable discovery-setup.service
-systemctl enable discovery-autostart.service
-systemctl enable discovery-register.service
```

Comment:
All good but discovery-register does not actually start on my instances:

```
[root@fdi ~]# systemctl status discovery-register
discovery-register.service - Register this host in Foreman
   Loaded: loaded (/etc/systemd/system/discovery-register.service; disabled)
   Active: inactive (dead)
```

I will merge this because I am currently working on TUI support which actually starts this service manually after TUI splash screen appears.